GameArena.java

```diff
@@ -466,8 +466,19 @@ public void removeText(Text t)
 	 */
 	public void pause()
 	{
-		try { Thread.sleep(20); }
-		catch (Exception e) {};
+		this.pause(20);
+	}
+
+	/**
+	 * 
+	 * Pause the {@link Thread} for {@param millis} milli-seconds
+	 * 
+	 * @param millis the number of milli-seconds the {@link Thread} is paused for
+	 */
+	public void pause(int millis) {
```

Comment:
would be better as a accessor/mutator pair to request and obtain a frame rate?

Reply:
Not quite sure what you mean by this?

Reply:
Oh okay, I understand what you mean now. Yeah that sounds interesting I can sort that out